pkg/pipeline/controller.go

```diff
@@ -215,6 +215,7 @@ func (c *Controller) Run(ctx context.Context) *livekit.EgressInfo {
 		select {
 		case <-c.stopped.Watch():
 			c.Info.Status = livekit.EgressStatus_EGRESS_ABORTED
+			c.Info.Error = "Start signal not received"
```

Comment:
would be great if we can dump any connection error strings from the template itself.